### PR TITLE
MARP-1079 Title on tab should include connector name

### PR DESCRIPTION
**Pull request by phhung-axonivy**

### CODE REVIEW

File: marketplace-ui/src/app/modules/product/product-detail/product-detail.component.ts [hunk 1]
```diff
@@ -130,7 +131,7 @@ export class ProductDetailComponent {
     this.updateDropdownSelection();
   }
 
-  constructor() {
+  constructor(private readonly titleService: Title) {
```
Comment (tvtphuc-axonivy) [inline marketplace-ui/src/app/modules/product/product-detail/product-detail.component.ts:134]: I think you missed this case 

image

File: marketplace-ui/src/app/modules/product/product-detail/product-detail.component.ts [hunk 2]
```diff
@@ -156,6 +157,7 @@ export class ProductDetailComponent {
         this.handleProductContentVersion();
         this.updateProductDetailActionType(productDetail);
         this.logoUrl = productDetail.logoUrl;
+        this.titleService.setTitle(productDetail.names[this.languageService.selectedLanguage()]);
```
Comment (tvtphuc-axonivy) [inline marketplace-ui/src/app/modules/product/product-detail/product-detail.component.ts:160]: Please use this.updateTitle();

File: marketplace-ui/src/app/modules/product/product-detail/product-detail.component.ts [hunk 3]
```diff
@@ -372,7 +374,13 @@ export class ProductDetailComponent {
     return tag;
   }
 
+  updateTitle() {
```
Comment (tvtphuc-axonivy) [inline marketplace-ui/src/app/modules/product/product-detail/product-detail.component.ts:377]: Should we change this to updateTitleForWebBrowser ?

File: marketplace-ui/src/app/app.routes.ts [hunk 4]
```diff
@@ -6,11 +6,13 @@ import { RedirectPageComponent } from './shared/components/redirect-page/redirec
 export const routes: Routes = [
   {
     path: 'error-page',
-    component: ErrorPageComponent
+    component: ErrorPageComponent,
+    title: 'Error Page'
```
Comment (tvtphuc-axonivy) [inline marketplace-ui/src/app/app.routes.ts:10]: please move 'Error Page' to constants

**Review by tvtphuc-axonivy — CHANGES_REQUESTED** [review]

File: marketplace-ui/src/app/modules/product/product-detail/product-detail.component.ts [hunk 5]
```diff
@@ -372,7 +374,13 @@ export class ProductDetailComponent {
...
+    const title = this.productDetail().names[this.languageService.selectedLanguage()];
```
Comment (tvtphuc-axonivy) [inline marketplace-ui/src/app/modules/product/product-detail/product-detail.component.ts:378]: I got error in console tab here 

image

File: marketplace-ui/src/app/modules/product/product-detail/product-detail.component.ts (hunk 1 above)
Comment (phhung-axonivy) [reply marketplace-ui/src/app/modules/product/product-detail/product-detail.component.ts:134]: This case is not a part of this story.
Could you create another various task to handle it?

**Review by tvtphuc-axonivy — APPROVED** [review]
LGTM

**Pull request merged**